bin/check-package-metadata.js

```diff
@@ -0,0 +1,149 @@
+#!/usr/bin/env node
+// Copyright IBM Corp. 2020. All Rights Reserved.
+// Node module: loopback-next
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+/**
+ * This is an internal script that will verify that all packages in our monorepo
+ * are registered in all required places and have the expected metadata in their
+ * package.json.
+ */
+'use strict';
+
+const path = require('path');
+const fs = require('fs-extra');
+
+const Project = require('@lerna/project');
+
+async function checkLicenseFiles(packages) {
+  for (const p of packages) {
+    if (!fs.existsSync(path.join(p.location, 'LICENSE'))) {
+      throw new Error(`${p.name} LICENSE file doesn't exist`);
+    }
+  }
+}
+
+async function checkMonorepoFile(packages, rootPath) {
+  const monorepoFile = await fs.readFile('docs/site/MONOREPO.md', 'utf8');
+
+  for (const p of packages) {
+    const packagePath = path.relative(rootPath, p.location);
+
+    const isPackageExist =
+      monorepoFile.includes(p.name) || monorepoFile.includes(packagePath);
+
+    if (!isPackageExist) {
+      throw new Error(
+        `MONOREPO.md doesn't contain mention about the ${p.name}`,
+      );
+    }
+  }
+}
+
+async function checkCodeOwnersFile(packages, rootPath) {
+  const excludes = ['@loopback/sandbox-example'];
+
+  const codeOwnersFile = await fs.readFile('CODEOWNERS', 'utf8');
+
+  for (const p of packages) {
+    if (excludes.includes(p.name)) {
+      continue;
+    }
+
+    const packagePath = path.relative(rootPath, p.location);
+
+    if (!codeOwnersFile.includes(packagePath)) {
+      throw new Error(
```

Comment:
Reporting violations via `throw new Error` is not very user friendly, because only the first error is reported and users have to repeatedly re-run the linter to discover all problems.

A better solution is to re-design the check functions to return either `undefined` on success or a list of error messages on failure.